game_eggs/minecraft/java/airplane/egg-airplane.json

```diff
@@ -0,0 +1,60 @@
+        "update_url": null
+    },
+    "exported_at": "2021-07-24T20:33:25+02:00",
+    "name": "Airplane",
+    "author": "info@ttr3.eu",
+    "description": "A stable, optimized and fast Paper fork.\r\nhttps:\/\/github.com\/TECHNOVE\/Airplane",
+    "features": null,
+    "images": [
+        "ghcr.io\/pterodactyl\/yolks:java_11",
+        "ghcr.io\/pterodactyl\/yolks:java_16"
+    ],
+    "file_denylist": [],
+    "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M  -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1 -Dusing.aikars.flags=https:\/\/mcflags.emc.gs -Daikars.new.flags=true -Dterminal.jline=false -Dterminal.ansi=true -jar {{SERVER_JARFILE}}",
+    "config": {
+        "files": "{\r\n    \"server.properties\": {\r\n        \"parser\": \"properties\",\r\n        \"find\": {\r\n            \"server-ip\": \"0.0.0.0\",\r\n            \"server-port\": \"{{server.build.default.port}}\"\r\n        }\r\n    }\r\n}",
+        "startup": "{\r\n    \"done\": \")! For help, type \",\r\n    \"userInteraction\": [\r\n        \"Go to eula.txt for more info.\"\r\n    ]\r\n}",
+        "logs": "{}",
+        "stop": "stop"
+    },
+    "scripts": {
+        "installation": {
+            "script": "apt update\r\napt -y install wget\r\ncd \/mnt\/server\r\nif [ -z \"$BUILDVERSION\" ] || [ \"$BUILDVERSION\" == \"latest\" ]; then\r\n    wget https:\/\/ci.tivy.ca\/job\/Airplane-${VER}\/lastSuccessfulBuild\/artifact\/launcher-airplane.jar -O ${SERVER_JARFILE}\r\nelse\r\n    wget https:\/\/ci.tivy.ca\/job\/Airplane-${VER}\/${BUILDVERSION}\/artifact\/launcher-airplane.jar -O ${SERVER_JARFILE}\r\nfi",
+            "container": "ubuntu",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Server file name",
+            "description": "Here you can set the server jar's name",
+            "env_variable": "SERVER_JARFILE",
+            "default_value": "server.jar",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:20"
+        },
+        {
+            "name": "Minecraft Version",
+            "description": "Here you can set the server's version (1.17 or 1.16)",
+            "env_variable": "VER",
+            "default_value": "1.17",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:20"
+        },
+        {
+            "name": "Build version",
+            "description": "Here you can set which build version you want to use from Airplane release",
```

Comment:
Would you please provide a URL from where users can find the build version `https://ci.tivy.ca`